packages/client/hmi-client/src/App.vue

```diff
@@ -7,7 +7,7 @@
 	<tera-navbar class="header" :active="displayNavBar" :show-suggestions="showSuggestions" />
 	<main>
 		<router-view v-slot="{ Component }">
-			<component class="page" ref="pageRef" :is="Component" :key="route.fullPath" />
```

Comment:
chaning this to path, the explorer would be remounted for every query change, we only need to have a unique key for the path without params